ibc/relayerfactory.go

```diff
@@ -29,6 +29,10 @@ type builtinRelayerFactory struct {
 	impl RelayerImplementation
 }
 
+func NewBuiltinRelayerFactory(impl RelayerImplementation) RelayerFactory {
```

Comment:
This probably should have just been an exported function in a previous PR. Needed for using simple relayers in `relayertest`.